### PR TITLE
fix istio resources not being in istio subfolder

### PR DESCRIPTION
**Pull request by jakobmoellersap**

### CODE REVIEW

File: operator/config/istio/virtual_service.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
+---
```
Comment (ruanxin) [inline operator/config/istio/virtual_service.yaml:2]:
```suggestion
```

File: operator/config/istio/virtual_service.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    - default/kyma-gateway
+  hosts:
+    - '*'
+  http:
+    - match:
+        - uri:
+            regex: \/v.*\/kyma\/event
+      route:
+        - destination:
+            host: operator-kyma-event-service
```
Comment (ruanxin) [inline operator/config/istio/virtual_service.yaml:17]:
```suggestion
            host: lifecycle-manager-kyma-event-service
```
This `operator-` prefix already changed, I think we also forgot to change it

File: operator/config/istio/virtual_service.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        - destination:
+            host: lifecycle-manager-kyma-event-service
+            port:
+              number: 8082
+    - match:
+        - uri:
+            regex: \/v.*\/manifest\/event
+      route:
+        - destination:
+            host: operator-manifest-event-service
```
Comment (ruanxin) [inline operator/config/istio/virtual_service.yaml:25]: who deploy this event service? module-manager( manifest-operator)? then the `operator-` prefix should also change.

File: operator/config/istio/virtual_service.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+            port:
+              number: 2022
```
Comment (jakobmoellersap) [inline operator/config/istio/virtual_service.yaml:27]:
```suggestion
```

**Review by ruanxin — APPROVED** [review]

**Pull request merged**